oxide.go

```diff
@@ -488,23 +475,26 @@ func (d *Driver) SetConfigFromFlags(opts drivers.DriverOptions) error {
 	}
 
 	if d.BootDiskImageID == "" {
-		err = errors.Join(err, fmt.Errorf("required option not set: %s", flagBootDiskImageID))
+		return errors.Join(err, fmt.Errorf("required option not set: %s", flagBootDiskImageID))
 	}
 
-	return errors.Unwrap(err)
+	return nil
```

Comment:
Since earlier the errors.Join is being used, I think this should be `return err` and line 478 should keep the `err = errors...`
I think the unwrap can be dropped though.

Reply:
Go generally expects explicit over implicit returns. The return at the end wouldn't necessarily be an error. Returning in the last error check explicitly ends the function where necessary.

Reply:
I missed this comment earlier. Fixed in https://github.com/oxidecomputer/rancher-machine-driver-oxide/commit/9544d64bfa097866686a1de4ec01054d948dc086.